azerothcore/azerothcore-wotlk
# DB/Quest: Assassin's Contract" & "Baron's Demise

- Fix for the Quest

Co-authored-by: Knindza <knindzagxg@users.noreply.github.com>





Fix for the Quest : Assasin's Contract & Baron's Demise

##### CHANGES PROPOSED:

-  Changed SAI, added coordinates for it
-  Added 4 Guards
- Assasin's will auto despawn after 60 sec when they re killed
- Respawn is diesbled until next event. 


###### ISSUES ADDRESSED:


Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1461


##### TESTS PERFORMED:

Sql is imported without issue. Tested in game with @Knindzagxg and everything is fine. P.S This quest didn't work before.


##### HOW TO TEST THE CHANGES:

1. .tele Southshore
2. Check if there is Assasin or Criiera
3. reload creature_template for every id
4. reload  smart_sccripts
5. restart and tele there again 
 [Video here :](https://www.youtube.com/watch?v=mz6AmNLBiak&feature=youtu.be)

##### KNOWN ISSUES AND TODO LIST:




##### Target branch(es):

Master





Big thanks for @Knindzagxg for fixing this event!!!
